cellrank/tl/estimators/_base_estimator.py

```diff
@@ -555,6 +532,60 @@ def compute_absorption_probabilities(
 
         self._write_absorption_probabilities(time=start, extra_msg=extra_msg)
 
+    def compute_lineage_priming(
+        self, method: str = "kl_divergence", early_cells: Optional[Sequence[str]] = None
+    ) -> pd.Series:
+        """
+        TODO.
```

Comment:
Suggestion: 

"Compute the degree of lineage priming. 

This method computes how naive vs. committed each individual cell is. It returns a score where 0 stands for naive and 1 stands for committed. "